playground/mocks/data/idp/idx/authenticator-enroll-data-phone.json

```diff
@@ -11,7 +11,7 @@
           "create-form"
         ],
         "name": "authenticator-enrollment-data",
-        "href": "http://localhost:3000/idp/idx/challenge",
```

Comment:
I think this mock was incorrect for its name (probably copied a verify mock) and should use the `/credential/enroll` href for enrollment